src/NServiceBus.AcceptanceTests/Routing/When_subscribing_to_a_derived_event_on_unicast_transports.cs

```diff
@@ -0,0 +1,110 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_subscribing_to_a_derived_event_on_unicast_transports : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Specific_event_should_be_delivered()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Publisher>(b => b.When(c => c.SubscriberSubscribed, async session =>
+                {
+                    await session.Publish(new SpecificEvent());
+                }))
+                .WithEndpoint<Subscriber>(b => b.When(async (session, c) => await session.Subscribe<SpecificEvent>()))
+                .Done(c => c.SubscriberGotEvent)
+                .Run();
+
+            Assert.True(context.SubscriberGotEvent);
+        }
+
+        [Test]
+        public async Task Base_event_should_not_be_delivered()
```

Comment:
After talking with Simon and Andreas we concluded this might make some sense in versioning scenarios.